app/assets/stylesheets/ltags/LiquidTags.scss

```diff
@@ -1,3 +1,4 @@
+@import 'CtaTag';
```

Comment:
Optional suggestion: This list looks like alphabetically arranged.

Reply:
✅